### PR TITLE
Fixed findbugs warnings, synchronized accesses in MQConnection.addMessageToQueue

### PR DESCRIPTION
**Pull request by TWestling**
Took care of some findbugs warnings that were added
years ago.
Findbugs was also upgraded due to a bug in 3.0.1.

### CODE REVIEW

**Comment by GLundh** [comment]
Lgtm, but you should probably mention the synchronization changes in the PR and commit message.

**Comment by TWestling** [comment]
Good point @GLundh , updated the PR and commit message.

**Review by GLundh — APPROVED** [review]

File: src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQConnection.java [hunk 1]
```diff
@@ -64,8 +64,8 @@
     private Connection connection = null;
     private Channel channel = null;
 
-    private static LinkedBlockingQueue messageQueue;
-    private static Thread messageQueueThread;
+    private volatile LinkedBlockingQueue messageQueue;
```
Comment (jonsten) [inline src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQConnection.java:67]: Not sure where we ended in this discussion, but volatile wasn't needed in the end, right?

**Review by jonsten — APPROVED** [review]

File: src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQConnection.java (hunk 1 above)
Comment (TWestling) [reply src/main/java/com/sonymobile/jenkins/plugins/mq/mqnotifier/MQConnection.java:67]: It wasn't when we talked about it, but removing it on the queue got me some more synchronization-related bugs from findbugs. I decided to go for good enough and leave it.

**Pull request merged**